test_infra/test_infra/databases_stack.py

```diff
@@ -0,0 +1,472 @@
+import json
+
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_glue as glue
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_kms as kms
+from aws_cdk import aws_rds as rds
+from aws_cdk import aws_redshift as redshift
+from aws_cdk import aws_s3 as s3
+from aws_cdk import aws_secretsmanager as ssm
+from aws_cdk import core as cdk
+
+
+class DatabasesStack(cdk.Stack):  # type: ignore
+    def __init__(
+        self,
+        scope: cdk.Construct,
+        construct_id: str,
+        vpc: ec2.IVpc,
+        bucket: s3.IBucket,
+        key: kms.Key,
+        **kwargs: str,
+    ) -> None:
+        """
+        AWS Data Wrangler Development Databases Infrastructure.
+        Includes Redshift, Aurora PostgreSQL, Aurora MySQL, Microsoft SQL Server.
+        """
+        super().__init__(scope, construct_id, **kwargs)
+
+        self.vpc = vpc
+        self.key = key
+        self.bucket = bucket
+
+        self._set_db_infra()
+        self._set_catalog_encryption()
+        self._setup_redshift()
+        self._setup_postgresql()
+        self._setup_mysql()
+        self._setup_sqlserver()
+
+    def _set_db_infra(self) -> None:
+        self.db_username = "test"
+        self.db_password = cdk.CfnParameter(self, "dbpassword", type="String").value_as_string
```

Comment:
no echo should be set to true like in the CFN [template](https://github.com/awslabs/aws-data-wrangler/blob/b13fcd8d169feb3219b4b4fff025dc6089cfe03b/cloudformation/databases.yaml#L8)

```cdk.CfnParameter(self, "dbpassword", type="String", no_echo=True)```

Link: https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.core/CfnParameter.html

Reply:
This should really be a secret instead, agree about the mask though.